app/controllers/api/auth/stories_controller.rb

```diff
@@ -47,7 +47,7 @@ def create_resource
       story.creator_id = current_user.id
       story.account_id ||= story.series.try(:account_id)
       story.account_id ||= current_user.account_id
-      story.account_id ||= current_user.approved_accounts.first.try(:id)
+      story.account_id ||= current_user.approved_accounts.first.try(:id) # not sure if I should change this to look at token
```

Comment:
So this is the fallback for the fallback (when there is neither a series.account nor a default user account), so I think it's fine to fallback on the first account, creating a story doesn't require :admin permissions.

There is another issue here though which is that the default account or series account may not be in the token's list of accounts, so we should check for that too - we shouldn't set the account to a value that the token does not include, does that make sense?